pkg/operator/resources/cluster/controller.go

```diff
@@ -60,7 +60,6 @@ func getControllerClusterPolicyRules() []rbacv1.PolicyRule {
 			Resources: []string{
 				"persistentvolumes",
 				"persistentvolumeclaims",
-				"volumesnapshots",
```

Comment:
Doesn't cdi-deployment need this?

Reply:
its listed under ```APIGroups: []string{
				"",
			},```

Reply:
ah okay 👍

Reply:
I see there is another rbac for "*" in "snapshot.storage.k8s.io"